src/frontend/apps/desk/.env.dev

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_URL=https://desk.127.0.0.1.nip.io/api/v1.0/
```

Comment:
I’m curious about how Next handles having both an `env.dev` and an `env.development` file. Is this addition necessary? We already define an environment variable when running the frontend in `env.d/dev/values.desk.yaml.gotmpl` at line 63

Reply:
Yes we don't need it.